packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -157,7 +173,7 @@ export const Combobox = <T extends string | number>(props: ComboboxProps<T>) =>
       return null;
     }
 
-    if (value === null) {
+    if (valueProp === undefined || valueProp === null) {
```

Comment:
I tried to create an `isNullish` function, but it didn't play well with Typescript.

Reply:
I can't remember where we go to with the whole empty string thing, but:

```ts
function isEmptyValue(value: unknown) {
  if (typeof value === 'string' && value === '') {
    return true;
  } else if (value === undefined || value === null) {
    return true;
  }

  return false;
}
```

Reply:
I think we may support `''`, because why not? If someone decides that they want to use that value, it's on them and will likely cause issues.